BP/localization/english/BP_decisions_l_english.yml

```diff
@@ -4,3 +4,35 @@
  select_hellenic_god_active_decision_desc:0 "#F Every devout [ROOT.Char.GetFaith.GetAdherentNameNoTooltip] should have kharis, or a devotion to a specific god or goddess who most strongly resonates with their soul.\n\nI should contemplate the various divine beings and decide which one calls to me the most.#!"
  select_hellenic_god_active_decision_confirm:0 "Contemplate kharis"
  select_hellenic_deity_decision_tt:0 "You gain a modifier associated with your selected kharis"
+ 
+ text_has_all_military_innovations_for_high_tribal:0 "Have the following Military Tribal [innovations|E]:\n$EFFECT_LIST_BULLET$$innovation_mustering_grounds$\n$EFFECT_LIST_BULLET$$innovation_castrae$"
+ text_has_all_civic_innovations_for_high_tribal:0 "Have the following Civic Tribal [innovations|E]:\n$EFFECT_LIST_BULLET$$innovation_development_00$\n$EFFECT_LIST_BULLET$$innovation_leges_barbarorum$\n$EFFECT_LIST_BULLET$$innovation_barter$\n$EFFECT_LIST_BULLET$$innovation_writing$\n$EFFECT_LIST_BULLET$$innovation_oppidum$"
+ text_has_entire_tribal_innovations:0 "Have all Tribal Era Military and Civic [innovations|E]"
+ text_has_entire_late_antiquity_innovations:0 "Have all Late Antiquity Era Military and Civic [innovations|E]"
+ text_has_required_early_migration_innovations:0 "Have the following Early Migraiton Era [innovations|E]:\n$EFFECT_LIST_BULLET$$innovation_migratory_tactics$\n$EFFECT_LIST_BULLET$$innovation_migratory_defenses$\n$EFFECT_LIST_BULLET$$innovation_double_entry_bookkeeping$\n$EFFECT_LIST_BULLET$$innovation_tenant_farming$\n$EFFECT_LIST_BULLET$$innovation_development_02$\n$EFFECT_LIST_BULLET$$innovation_doctrine_and_dogma$"
+ text_has_entire_early_migration_innovations:0 "Have all the Early Migration Era Military and Civic [innovations|E]"
+ 
+ convert_to_high_tribal_decision:0 "Convert to High Tribal Government"
+ convert_to_high_tribal_decision_desc:0 "We abandon our tribal ways and embrace a more advanced form of government, capable of meeting the demands of the current era."
+ convert_to_high_tribal_decision_confirm:0 "Confirm"
+ 
+ convert_to_subroman_decision:0 "Convert to Subroman Government"
+ convert_to_subroman_decision_desc:0 "We abandon our tribal ways and embrace, to the best of our ability, the ways of our Roman ancestors."
+ convert_to_subroman_decision_tooltip:0 "Change [government|E] type by becoming a Sub-Roman government"
+ convert_to_subroman_decision_confirm:0 "Adopt the Faded Roman Institutions"
+ 
+ convert_to_bureaucratic_decision:0 "Convert to Bureacratic Government"
+ convert_to_bureaucratic_decision_desc:0 "Descended from the great Romans, we will embrace and reinvigorated their ways, adopting the customs, mannerism, and goverance of Imperial Rome."
+ convert_to_bureaucratic_decision_tooltip:0 "Change [government|E] type by becoming a Sub-Roman government"
```

Comment:
I think this is a mistake.

Reply:
Yeah, that's a typo that needs to be fixed.